src/components/tx-flow/flows/SignMessage/SignMessage.tsx

```diff
@@ -175,9 +130,9 @@ export type ConfirmProps = BaseProps & {
   requestId?: RequestId
 }
 
-const SignMessage = ({ onClose, message, safeAppId, requestId }: ProposeProps | ConfirmProps): ReactElement => {
+const SignMessage = ({ message, safeAppId, requestId }: ProposeProps | ConfirmProps): ReactElement => {
   // Hooks & variables
-  const [showCloseTooltip, setShowCloseTooltip] = useState<boolean>(false)
+  const { onClose } = useContext(TxModalContext)
```

Comment:
How is this used? It was supposed to be an internal callback that outside components should only assign, but not call. Only TxModalProvider should call it. It shouldn't even be exported (my mistake that it is).

Reply:
When the user submits the [last sync signature](https://github.com/safe-global/safe-wallet-web/blob/dev/src/hooks/messages/useSyncSafeMessageSigner.ts#L37-L57) or it is received [via poll](https://github.com/safe-global/safe-wallet-web/blob/dev/src/hooks/messages/useSyncSafeMessageSigner.ts#L21-L25), it [closes the modal](https://github.com/safe-global/safe-wallet-web/blob/dev/src/services/safe-messages/safeMsgNotifications.ts#L34-L40).

Reply:
In this scenario you’re supposed to call setTxFlow(undefined).

Reply:
Good point. I've updated it accordingly, as well as removing `onClose` from the context.